### PR TITLE
Create join-ornek1.txt

### PR DESCRIPTION
**Pull request by nurdancetinkaya**
ı added extensive and basic example for join.

### CODE REVIEW

**Comment by alperyilmaz** [comment]
The example is too long and redundant. The sample files should be designed so that every aspect of join can be shown on them..

File: code/join-ornek1.txt [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+c London
+d Toronto
+
+$ join testfile1 testfile2
+a India NewDelhi
+b US Washington
+c UK London
+d Canada Toronto
+
+$ join -a1 testfile1 testfile2
```
Comment (alperyilmaz) [inline code/join-ornek1.txt:69]: countries and capitals is nice example.. we can prepare data files of population, unemployment rate, powerty and then we can ask many questions on them

**Comment by nurdancetinkaya** [comment]
Hocam merhaba, bazı ülkeler için dediğiniz bilgilerle alakalı aşağıdaki dosyaları oluşturdum.Bu dosyalar üzerinden join kullanımıyla ilgili şimdi soru düşünsem olurmu?

$ cat country.txt
 Country Capital Population Unemployment Rate Poverty(%)
 Russia Moscow 146,727,405 5.40 11.0
 Germany Berlin 82,175,700 4.10 15.5
 Turkey Ankara 78,741,053 11.30 16.9
 Thailand Bangkok 68,145,000 1.20 13.2
 France Paris 66,821,000 10.00 7.9
 UK Londra   65,110,000 4.80 16.2
 Italy Roma 60,592,547 11.70 29.9
 Saudi Arabia Riyad 32,248,000 5.40 19.5

 $ cat country.txt
 Country Capital
 Russia Moscow
 Germany Berlin
 Turkey Ankara
 Thailand Bangkok
 France Paris
 UK Londra
 Italy Roma
 Saudi Arabia

 $ cat population.txt
 Country Capital Population
 Russia Moscow 146,727,405
 Germany Berlin 82,175,700
 Turkey Ankara 78,741,053
 Thailand Bangkok 68,145,000
 France Paris 66,821,000
 UK Londra   65,110,000
 Italy Roma 60,592,547
 Saudi Arabia Riyad 32,248,000

 $ cat unemployment.txt
 Country Capital Unemployment(%)
 Russia Moscow 5.40
 Germany Berlin 4.10
 Turkey Ankara 11.30
 Thailand Bangkok 1.20
 France Paris 10.00
 UK Londra 4.80
 Italy Roma 11.70
 Saudi Arabia Riyad 5.40

 $ cat poverty.txt
 Country Capital  Rate Poverty(%)
 Russia Moscow 11.0
 Germany Berlin 15.5
 Turkey Ankara 16.9
 Thailand Bangkok  13.2
 France Paris 7.9
 UK Londra 16.2
 Italy Roma 29.9
 Saudi Arabia Riyad 19.5








________________________________
Gönderen: Alper Yilmaz <notifications@github.com>
Gönderildi: 26 Kasım 2016 Cumartesi 12:45
Kime: alperyilmaz/BYM4591-book
Bilgi: nurdancetinkaya; Author
Konu: Re: [alperyilmaz/BYM4591-book] Create join-ornek1.txt (#62)


@alperyilmaz commented on this pull request.

________________________________

In code/join-ornek1.txt<https://github.com/alperyilmaz/BYM4591-book/pull/62#pullrequestreview-10212855>:

> +d Canada
+e Australia
+
+$ cat testfile2
+a NewDelhi
+b Washington
+c London
+d Toronto
+
+$ join testfile1 testfile2
+a India NewDelhi
+b US Washington
+c UK London
+d Canada Toronto
+
+$ join -a1 testfile1 testfile2


countries and capitals is nice example.. we can prepare data files of population, unemployment rate, powerty and then we can ask many questions on them

-

**Comment by alperyilmaz** [comment]
closing.. same reason as in #63

**Pull request closed without merging**